openerp/addons/base/tests/test_xmlrpc.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 import openerp.tests.common
 
-DB = openerp.tests.common.DB
+DB = openerp.tests.common.get_db_name()
```

Comment:
should probably call the function in `TestCase.setUp` or each callsite rather than store its result in a constant